main.js

```diff
@@ -449,7 +450,26 @@
                     fetchWPRDCData(dataSourceName);
                 });
                 retryDiv.appendChild(retryButton);
-            }));
+            }))
+            .then(() => {
+                return fetch(WPRDC_META_URL + dataSource.id)
+                .then((response) => {
+                    //check response for data and data for date
+                    if (response.status >= 200 && response.status < 300) {
+                        return response.json();
+                    }
+                })
+                .then((metadata) => {
+                    // Parse json for last modified field
+                    var parsedDate = Date.parse(metadata.result.last_modified);
+
+                    // Display a notification if the dataset has been updated within the hour
+                    var diff = Date.now() - parsedDate;
+                    if (diff <= 3600000) {
```

Comment:
Sure thing - also an hour seems pretty short, I've never actually gotten an notification for it in testing (but it definitely works for longer periods of time). So I might up the time from 1 hour to 2 or 3 if that's cool